### PR TITLE
fixed https://github.com/NuGet/Home/issues/1163

### PR DESCRIPTION
**Pull request by zhili1208**
https://github.com/NuGet/Home/issues/1163
OutputConsole will check msbuild verbosity setting, if the message is error or msbuild verbosity setting is greater than current message level, then show the message

### CODE REVIEW

File: src/VsExtension/OutputConsoleLogger.cs [hunk 1]
```diff
@@ -40,6 +42,9 @@ public OutputConsoleLogger(IServiceProvider serviceProvider)
             _buildEvents.OnBuildBegin += (obj, ev) => { ErrorListProvider.Tasks.Clear(); };
             _solutionEvents = dte.Events.SolutionEvents;
             _solutionEvents.AfterClosing += () => { ErrorListProvider.Tasks.Clear(); };
+            var properties = dte.get_Properties("Environment", "ProjectsAndSolution");
+            var value = properties.Item("MSBuildOutputVerbosity").Value;
+            _msBuildOutputVerbosity = value is int ? (int)value : 0;
```
Comment (yishaigalatzer) [inline src/VsExtension/OutputConsoleLogger.cs]: What happens if the user changes the logging settings while vs is running? Is this value cached forever here?

File: src/VsExtension/OutputConsoleLogger.cs [hunk 2]
```diff
@@ -58,7 +63,11 @@ public void End()
         public void Log(MessageLevel level, string message, params object[] args)
         {
             var s = string.Format(CultureInfo.CurrentCulture, message, args);
-            OutputConsole.WriteLine(s);
+
+            if ((int)level < _msBuildOutputVerbosity || level == MessageLevel.Error)
```
Comment (yishaigalatzer) [inline src/VsExtension/OutputConsoleLogger.cs]: why not keep _msBuildOutputVerbosity as an Enum ?

**Comment by yishaigalatzer** [comment]
Are we able to have a functional test here?

:watch:

**Comment by emgarten** [comment]
This should not use msbuild verbosity, it's not related for UI install scenarios.
:watch:

**Comment by emgarten** [comment]
:shipit:

**Comment by deepakaravindr** [comment]
:shipit:

**Pull request closed without merging**